Makefile

```diff
@@ -160,3 +161,6 @@ build-image-tnf:
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${IMAGE_TAG} \
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${TNF_VERSION} \
 		-f Dockerfile .
+classification-js:
+	./tnf generate catalog javaScript > script/classification.js
```

Comment:
File generated by this line is not working in the page and not matching with the script/classification.js in this PR